src/test/java/io/github/sskorol/converters/StringToLocalDateConverter.java

```diff
@@ -0,0 +1,13 @@
+package io.github.sskorol.converters;
+
+import com.creditdatamw.zerocell.converter.Converter;
+
+import java.time.LocalDate;
+
+public class StringToLocalDateConverter implements Converter<LocalDate> {
```

Comment:
There's and existing [LocalDateConverter](https://github.com/creditdatamw/zerocell/blob/master/core/src/main/java/com/creditdatamw/zerocell/converter/LocalDateConverter.java#L8) in ZeroCell.

Reply:
My bad. They have added the default converters in the latest version. I was using older version of zerocell. Will fix these converters issues.